README.md

```diff
@@ -61,7 +61,7 @@ Sample models are available [here](https://github.com/luxonis/depthai/tree/main/
     Note: `colcon build` will also work with ROS1
 1. Run the Nodelet
     ```bash
-    ros2 run depthai_ros_driver depthai_ros_driver_node
+    ros2 run depthai_ros_driver depthai_ros_driver_depthai_ros_driver_node
```

Comment:
This seems bad. If we are prefixing package name, we should improve the name used in the cmake file.

Thoughts?

Reply:
Agree, i was thinking the same too. I can rename the {name} in the cmakelist. Output:
`depthai_ros_driver_sensor_driver_node`

Or edit the macro, remove the use of {name} and substitute it with {project_name}. Back to `depthai_ros_driver_node`

Reply:
I noticed we didn't use the set_target_property for OUTPUT_NAME. Is that no longer requried?

I remember that being essential to prevent name conflicts (hence the original suggestion).

If it is not required anymore, we can switch back to not adding ${PROJECT_NAME} to the target_names

Reply:
the exe is located in `lib/{PROJECT_NAME}/{NAME}_node`. so I reverted it back to `depthai_ros_driver_node`

Reply:
@gbiggs PTAL at my comment above. If the current change is ok (no conflicts with different nodes having the same name, then the PR LGTM :)

Reply:
The node library and node executable are both placed in a directory named after the package location. So long as two nodes with the same name exist in separate packages, there shouldn't be any conflicts in usage.

Have you noticed a case where conflicts occur in the past?

Reply:
I can't find the link, but I remember, a while ago, the best practice was to prefix the project_name and then rename the output_name property.

Perhaps, it's changed. In that case, no issues

Reply:
Noted :+1:

Reply:
I just found the reference. It's noted in the default cmakelists file created by catkin_create_pkg

Reply:
Perfect then. :) Thanks a lot

LGTM for merge